analyses/ewings-cell-type/references/tumor_marker_genes.tsv

```diff
@@ -0,0 +1,12 @@
+cell_type	gene_symbol	ensembl_gene_id	source
+tumor	CD99	ENSG00000002586	10.1158/2767-9764.CRC-23-0027
```

Comment:
In looking at the references, `CD99` was also found in 10.1101/2024.01.18.576251? I don't know if you want to have multiple sources on each line, or repeated lines, but I think you should probably capture when a gene is supported by more than one reference somehow. (I'd probably do the tidier repeated lines with 1 ref per line, and then deduplicate in code)

Update: I see now where you got this specific list, which does not include CD99, though there is a slightly later sentence:
> We found that compared to other cell types in the primary tumor, Ewing cells across all samples express moderate to high levels of PAPPA but low levels of most HLA genes. For immunotherapeutic targets, Ewing sarcoma cells uniquely express higher levels of ADGRG2, IL1RAP, and LINGO1. CD99 and STEAP1 is also highly expressed in Ewing sarcoma cells, although there is significant expression in CAFs as well.

I don't know if you want to include any of those extra genes too? There is also a longer list in Supp table 2, but I don't understand how that list was reduced to the lists in text

Reply:
From 10.1101/2024.01.18.576251: "Among the Ewing sarcoma cells, a proliferating population was identified by the expression of MKI67 and PCNA (Fig. 1C-D)." Would you want to include such subtyping?

Reply:
> Update: I see now where you got this specific list, which does not include CD99, though there is a slightly later sentence:

So CD99 has been the historic marker of Ewing's cells, but it is also expressed at a slightly lower level in normal cells. Fig 1F in 10.1101/2024.01.18.576251 shows that in this particular set of patients its expressed at varying levels across all cells. I got the list I used for this paper specifically from Fig 1C because there are a lot of sentences in the text that mention various genes that were used. However, looking at 1F a lot of those genes are either only in a subset of patients. 

> I don't know if you want to include any of those extra genes too? There is also a longer list in Supp table 2, but I don't understand how that list was reduced to the lists in text

I'm trying to keep this initial list relatively short and only stick to genes that are consistently expressed across multiple patients. I know this is hard because the dataset we have is probably going to have differences, but I want to use CellAssign and the longer the list the more resources that takes. Looking at Figure 1F, maybe we add in ADGRG2, but LINGO1 and IL1RAP are not consistently expressed. 

> From 10.1101/2024.01.18.576251: "Among the Ewing sarcoma cells, a proliferating population was identified by the expression of MKI67 and PCNA (Fig. 1C-D)." Would you want to include such subtyping?

Yes definitely! In this initial list I just wanted to do things that marked tumor cells, but then we should create a list that marks tumor cell states. There are a lot more cell states that have been identified besides this, so that will be a more comprehensive list and I think should be part of a separate review.

Reply:
This all makes sense! I wrote all that as I was reviewing, and I agree that a lot of it is for potential future directions. I agree especially with keeping the initial list compact. 

As I said in the overall comments, the main thing is to have some documentation of the decisions in the repo itself!